onadata/apps/api/viewsets/xform_list_api.py

```diff
@@ -113,6 +121,9 @@ def filter_queryset(self, queryset):
     def list(self, request, *args, **kwargs):
         self.object_list = self.filter_queryset(self.get_queryset())
 
+        if request.method == 'HEAD':
+            return self.get_response_for_head_request()
```

Comment:
it feels like a waste to do this after making significant database queries, but i wanted this PR to have a narrow focus, and i didn't want to rewrite authentication-checking logic